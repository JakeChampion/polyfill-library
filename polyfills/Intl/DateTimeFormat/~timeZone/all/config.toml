#### Dependencies
dependencies = [
  "Intl.DateTimeFormat"
]

#### Specification Link
spec = "https://tc39.es/ecma402/#sec-initializedatetimeformat"

#### Documentation Link
docs = "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#using_options"

#### License
license = "MIT"

#### Notes
notes = [
  "See formatjs documentation for more info on 'golden' vs 'all' timezones. https://formatjs.io/docs/polyfills/intl-datetimeformat"
]

#### Browser compatibility
[browsers]
android = "<4.4"
ie = "9 - *"
ie_mob = "9 - *"
firefox = "<52"
opera = "<15"
chrome = "<24"
safari = "<10"
ios_saf = "<10"
firefox_mob = "<52"
samsung_mob = "<1.5"

[install]
module = "@formatjs/intl-datetimeformat"
paths = [ "/add-all-tz.js" ]
clean = [
  "polyfill.js",
]
