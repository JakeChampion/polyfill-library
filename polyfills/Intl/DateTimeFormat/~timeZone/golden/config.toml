#### Dependencies
dependencies = [
  "Intl.DateTimeFormat"
]

#### Specification Link
spec = "https://tc39.es/ecma402/#sec-initializedatetimeformat"

#### Documentation Link
docs = "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#using_options"

#### License
license = "MIT"

#### Notes
notes = [
  "See formatjs documentation for more info on 'golden' vs 'all' timezones. https://formatjs.io/docs/polyfills/intl-datetimeformat"
]

#### Browser compatibility
# Browser compatibility list sourced from Intl.DateTimeFormat.
[browsers]
android = "*"
ie = "9 - *"
ie_mob = "9 - *"
edge = "<79"
edge_mob = "<79"
firefox = "*"
opera = "*"
chrome = "<76"
safari = "*"
ios_saf = "*"
firefox_mob = "*"
samsung_mob = "*"

[install]
module = "@formatjs/intl-datetimeformat"
paths = [ "/add-golden-tz.js" ]
clean = [
  "polyfill.js",
]
